site/content/docs/cicd/deployment-settings-file/ASP.NET Core App to AWS App Runner.md

```diff
@@ -0,0 +1,133 @@
+**Recipe ID:** AspNetAppAppRunner
+
+**Recipe Description:** This ASP.NET Core application will be built as a container image on Linux and deployed to AWS App Runner, a fully managed service for web applications and APIs. If your project does not contain a Dockerfile, it will be automatically generated, otherwise an existing Dockerfile will be used. Recommended if you want to deploy your web application as a Linux container image on a fully managed environment.
+
+**Settings:**
```

Comment:
Since we are generating this documents, I am unsure how this snippet would look like in our case. Our top level settings don't have a defined object that the user can reference to know what that looks like. Our navigation table is per recipe. Generating this snippet with all the settings and nested objects under it (or just specifying the object name and then creating another document that defines that shape like CloudFormation does) requires a scope increase. While I do agree that this could be beneficial, I prefer to flush out the details with the team and tackle it as additional work.

Reply:
Love this idea of showing a sample JSON settings object but agree we could do that as a separate story.